cmd/single-ocsp/main.go

```diff
@@ -26,7 +26,7 @@ type PKCS11Config struct {
 	TokenLabel      string
 	PrivateKeyLabel string
 	PIN             string
-	SlotID          float64
+	SlotID          uint
```

Comment:
I was arguing for int over uint. I don't think float64 is right.

That said, see my comment in https://github.com/letsencrypt/boulder/pull/784#issuecomment-139811009. I think selecting a token by slot id may be an incorrect usage of the PKCS11 API, and we should instead use just the token label.

I'm not super familiar with PKCS11, but there are a few reasons I think this:
 - The slot id's type is actually `CK_SLOT_ID`, not `uint`. It happens to be the case that it's a uint, but that may be an implementation detail.
 - The module I am using fails if you call `GetTokenInfo` for a given `CK_SLOT_ID` without calling `GetSlotInfo` first.
 - The values of `CK_SLOT_ID` are small integers, which strongly suggests they may change when there are multiple PKCS11 tokens attached, based on attachment order. I don't have multiple tokens to test with, though.
 - This excerpt from ftp://ftp.rsasecurity.com/pub/pkcs/pkcs-11/v2-30/pkcs-11v2-30b-d6.pdf does in fact suggest side effects of `GetSlotInfo`, even though it doesn't directly address the one we're talking about.

> All slots which C_GetSlotList reports must be able to be queried as valid slots by C_GetSlotInfo.  Furthermore, the set of slots accessible through a Cryptoki library is checked at the time that C_GetSlotList, for list length prediction (NULL pSlotList argument) is called. If an application calls C_GetSlotList with a non-NULL pSlotList, and then the user adds or removes a hardware device, the changed slot list will only be visible and effective if C_GetSlotList is called again with NULL. Even if C_GetSlotList is successfully called this way, it may or may not be the case that the changed slot list will be successfully recognized depending on the library implementation.  On some platforms, or earlier PKCS11 compliant libraries, it may be necessary to successfully call C_Initialize or to restart the entire system.

So, I think my current preferred approach is to drop the slot ID from the config and just use the token label, but I'm interested in input from folks more familiar with PKCS11.